### ISSUE TITLE
Differences between Jackett and Prowlarr YAML definitions

### ISSUE THREAD

**Issue by ilike2burnthing** [Prowlarr/Indexers#370]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Posting this mainly for clarity on my end, not sure if it will necessarily be useful for others. Doesn't have to stay open, just posting as a record and in case I've misunderstood anything.

I've just gone through all definitions and submitted PRs where needed, so below should be exhaustive.

---

Prowlarr has 4 additional definitions:
- bjshare - there's a lot different compared with Jackett's C# indexer, but no maintainers have an account to work on it
- ncore - same ^
- brasiltracker - needs rewritten in C# - https://github.com/Prowlarr/Indexers/issues/336#issuecomment-1548841164
- xthor-scraping - definition seems to need some work, but no active Jackett maintainers have an account

---

Common minor differences:
- info notes - `Jackett` changed to `Prowlarr`
- login - `inputs: {} # TODO: remove in v10`

---

Other differences:
- xxxclub & rgfootball - UA header needed for Prowlarr but not Jackett
- showrss - id in Prowlarr is `showrss-yml` as there is a native showrss indexer
- torrentleech - IMDB TV search removed (https://github.com/Prowlarr/Indexers/issues/135) and exclude archives setting not added (https://github.com/Jackett/Jackett/commit/93a26a78f5dce0208ee0f10440c48606939fd78d) as json filters not supported(?)
- yggtorrent/cookie - category search not added (https://github.com/Jackett/Jackett/pull/14250) as not supported(?)

### Expected Behavior

N/A

### Steps To Reproduce

N/A

### Environment

```markdown
N/A
```

### What branch are you running?

Master

### Trace Logs?

N/A

**Comment by JigSawFr** [comment]
@ilike2burnthing can you tell me what we need to do on xthor-scrapping ?

**Comment by ilike2burnthing** [comment]
Just comparing the 2 indexers:
- can tmdbid search be used?
- multilang, multilanguage, and vostfr settings and title filters can be copied across from the API indexer
- are posters available?
- date is missing
- MST and MR are missing
- engine tag is missing (even if it's just `# engine n/a`)

**Comment by Hannah-GBS** [comment]
For torrentleech's exclude archives feature, jackett has just changed that to use the new `-tags:` syntax Torrentleech recently implemented, so that should be able to be brought forward to Prowlarr now I believe

https://github.com/Jackett/Jackett/commit/09a67a50f8067ea1de7b0722111cede546aab125

**Comment by bakerboy448** [comment]
@ilike2burnthing at your convenience can you re-visit and update this :)

**Comment by ilike2burnthing** [comment]
See https://github.com/Prowlarr/Indexers/pull/466.

Also https://github.com/Prowlarr/Indexers/pull/432, but requires C# indexer be removed as well.

Some updates to OP ^

**Comment by ilike2burnthing** [comment]
@bakerboy448 yes, there was a difference between Jackett and Prowlarr's YGG indexers, as Prowlarr doesn't have support for selecting a search path based on the chosen categories - https://github.com/Jackett/Jackett/commit/abca0e91899ec9217d319cfc404b65f028f83a94

So Prowlarr instead used a setting for choosing categories and subcategories - https://github.com/Prowlarr/Indexers/commit/c4a8c41cbe7b4cf5bb85d797461b0ad2b6882a74

**Comment by naXa777** [comment]
for completeness please add a word about U2P removal.

- https://github.com/Prowlarr/Indexers/pull/814

**Comment by AlexanderFeyaerts** [comment]
torrent-heaven does not seem to work on prowlarr, but does on jackett.

**Comment by ilike2burnthing** [comment]
https://discord.com/channels/264387956343570434/1443682810207666186
